.circleci/config.yml

```diff
@@ -32,15 +32,20 @@ jobs:
           version: 0.16.0
       - heroku-buildpacks/install-build-dependencies
       - run:
-          name: "Build and package buildpack"
+          name: "Build and package buildpack with retries"
           command: |
-            package_toml="<< parameters.buildpack-dir >>/package.toml"
-            if [[ -f "<< parameters.buildpack-dir >>/build.sh" ]]; then
-              "./<< parameters.buildpack-dir >>/build.sh"
-              package_toml="<< parameters.buildpack-dir >>/target/package.toml"
-            fi
+            n=1
```

Comment:
Is this to work around ECR download failures?

Reply:
yes, I made a similar change here: https://github.com/heroku/pack-images/pull/156